python/lsst/afw/display/virtualDevice.py

```diff
@@ -41,7 +41,7 @@ def __del__(self):
 
     def _close(self):
         """!Close the display, cleaning up any allocated resources"""
-        if self.verbose:
+        if hasattr(self, "verbose") and self.verbose and hasattr(self, "display"):
```

Comment:
It's because the attributes are not created if the object `__init__` has not been run.